src/services/EdFi.DataManagementService.Api.Tests/Core/Middleware/ValidateDocumentMiddlewareTests.cs

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Text.Json.Nodes;
+using EdFi.DataManagementService.Api.ApiSchema;
+using EdFi.DataManagementService.Api.Core.Middleware;
+using EdFi.DataManagementService.Api.Core.Model;
+using EdFi.DataManagementService.Api.Core.Validation;
+using EdFi.DataManagementService.Core.Pipeline;
+using FluentAssertions;
+using Json.Schema;
+using Microsoft.Extensions.Logging.Abstractions;
+using NUnit.Framework;
+
+namespace EdFi.DataManagementService.Api.Tests.Core.Middleware;
```

Comment:
Just a note that Andony moved this namespace around on us to EdFi.DataManagementService.Api.Tests.Unit.Core.Middleware